packages/backend/src/payment-method/ilp/peer/model.ts

```diff
@@ -68,7 +68,8 @@ export class Peer
             },
             liquidityThreshold: this.liquidityThreshold,
             balance
-          }
+          },
+          tenantId // TODO: update when peers are tenanted
```

Comment:
For now this is using the operator tenant ID, but will be replaced with the peer's own tenant id once https://github.com/interledger/rafiki/issues/3129 is merged in.